prog/main.go

```diff
@@ -176,6 +177,7 @@ func main() {
 	flag.StringVar(&flags.app.s3URL, "app.collector.s3", "local", "S3 URL to use (when collector is dynamodb)")
 	flag.StringVar(&flags.app.controlRouterURL, "app.control.router", "local", "Control router to use (local or sqs)")
 	flag.StringVar(&flags.app.pipeRouterURL, "app.pipe.router", "local", "Pipe router to use (local)")
+	flag.StringVar(&flags.app.natsHostname, "app.nats", "", "Hostname for NATS")
```

Comment:
What's the app behavior if `natsHostname` is not provided? Probably should document that here in the option.